feat(chair): send email on saved schedule

jhaimec opened the pull request.

jhaimec committed:
feat(chair): send email on saved schedule

jhaimec committed:
fix(chair): not sending email on schedule save

hijacque reviewed (changes requested):
move feature to when posting faculty schedule
meron pang isang function `postFacultySchedules()`, after masave lahat ng schedules dun palang pwede mapost para maview na ng faculty sa department

hijacque committed:
fix(chair): send email notif of posted schedule

- send multiple email to all faculty in the department informing on the posted schedules
additional: moved alert message for unsaved faculty schedule with incomplete plotting
